ci_support/run_docker_build.sh

```diff
@@ -37,8 +37,6 @@ echo "$config" > ~/.condarc
 # A lock sometimes occurs with incomplete builds. The lock file is stored in build_artefacts.
 conda clean --lock
 
-# library dependencies for matplotlib
-yum install -y libXext libXrender libSM tk libX11-devel
```

Comment:
This line prevents us merging this PR. I'm going to start looking into this once https://github.com/conda-forge/conda-forge.github.io/issues/35 has been solved.